internal/vehicle/mercedes.go

```diff
@@ -0,0 +1,74 @@
+package vehicle
+
+import (
+	"errors"
+	"strings"
+	"time"
+
+	"github.com/andig/evcc/api"
+	"github.com/andig/evcc/internal/vehicle/mercedes"
+	"github.com/andig/evcc/util"
+	"golang.org/x/oauth2"
+)
+
+// Mercedes is an api.Vehicle implementation for Mercedes cars
+type Mercedes struct {
+	*embed
+	*mercedes.Identity
+	*mercedes.Provider
+}
+
+func init() {
+	registry.Add("mercedes", NewMercedesFromConfig)
+}
+
+// NewMercedesFromConfig creates a new Mercedes vehicle
+func NewMercedesFromConfig(other map[string]interface{}) (api.Vehicle, error) {
+	cc := struct {
+		Title                  string
```

Comment:
See updated embed handling in other cars

Reply:
Will be updated after the branch is updated with the master.